restli-server/src/main/java/com/linkedin/restli/server/config/RestLiMethodConfigImpl.java

```diff
@@ -24,7 +26,8 @@ public boolean shouldValidateQueryParams() {
     return _validateQueryParams;
   }
 
-  public void setValidateQueryParams(boolean validateQueryParams) {
```

Comment:
This is a public method, you shouldn't remove it. If it seems useless, then please just deprecate it.

Reply:
+1

Reply:
kept as it is